### PR TITLE
Fix adding users via pairing code

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-13428
affects: parent
release note: Fixed adding children with pairing code

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/Core/Views/AlertAction.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import UIKit
+
+public class AlertAction: UIAlertAction {
+    public var handler: ((UIAlertAction) -> Void)?
```
Comment (bluwave) [inline Core/Core/Views/AlertAction.swift]: should this be weak.  There are no weird retain issues with this?

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1009173&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed adding children with pairing code

#### Affected Apps: Parent

[MBL-13428](https://instructure.atlassian.net/browse/MBL-13428)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.24% | 93.23% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Views/AlertAction.swift (hunk 1 above)
Comment (armstrongnate) [reply Core/Core/Views/AlertAction.swift]: Swift won't let me apply `weak` to a callback.

```
'weak' may only be applied to class and class-bound protocol types
```

**Pull request merged**